django_project/frontend/src/pages/Admin/Basemap/List/index.jsx

```diff
@@ -34,6 +36,13 @@ export function resourceActions(params) {
  * Indicator List App
  */
 export default function BasemapList() {
+  const [searchParams, setSearchParams] = useSearchParams();
+
+  let defaultFilter = JSON.parse(window.sessionStorage.getItem(urls.api.list, "{}"))
```

Comment:
We don't need to use session storage, just parse from url
Like below
https://github.com/unicef-drp/GeoSight-OS/blob/main/django_project/frontend/src/pages/Admin/DataBrowser/index.jsx#L65